src/com/lftechnology/java/training/sanish/printday/Days.java

```diff
@@ -0,0 +1,58 @@
+package com.lftechnology.java.training.sanish.printday;
+
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Print days as give day index
+ * 
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ *
+ */
+public class Days {
+	private static final Logger LOGGER = Logger.getLogger(Days.class.getName());
+	private static final String SUNDAY = "Sunday";
+	private static final String MONDAY = "Monday";
+	private static final String TUESDAY = "Tuesday";
+	private static final String WEDNESDAY = "Wednesday";
+	private static final String THURSDAY = "Thusday";
+	private static final String FRIDAY = "Friday";
+	private static final String SATURDAY = "Saturday";
+	private static final String INVALID_DAY = "Not a valid day";
+	private static final String[] DAYS = { SUNDAY, MONDAY, TUESDAY, WEDNESDAY,
+			THURSDAY, FRIDAY, SATURDAY };
```

Comment:
its better if you create a separate class like `CommonConstants` for all these contants..